.github/workflows/main.yml

```diff
@@ -0,0 +1,73 @@
+name: Uradhura
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+    inputs:
+      job:
+        description: 'Deploy manually'
+        required: true
+        default: 'deploy'
+
+env:
+  CARGO_TERM_COLOR: always
+  PROJECT_NAME: ${{ github.event.repository.name }}
+  SSH_USER: ${{ secrets.SSH_USER }}
+  SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
+  DEPLOYMENT_HOST: ${{ secrets.DEPLOYMENT_HOST }}
+
+jobs:
+  # build:
+  #   runs-on: ubuntu-latest
+
+  #   steps:
+  #     - name: Checkout
+  #       uses: actions/checkout@v1
+
+  #     - name: Install latest rust toolchain
+  #       uses: actions-rs/toolchain@v1
+  #       with:
+  #         toolchain: stable
+  #         default: true
+  #         override: true
+
+  #     - name: Build
+  #       run: cargo build --all --release && strip target/release/$PROJECT_NAME
+
+  # release:    
+  #   runs-on: ubuntu-latest
+
+  #   steps:
+  #     - name: Release
+  #       uses: softprops/action-gh-release@v1
+  #       # if: startsWith(github.ref, 'refs/tags/')
+  #       with:
+  #         files: |
+  #           target/release/$PROJECT_NAME_amd64
+  #       env:
+  #         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: add host key verification
+        run: |
+          eval $(ssh-agent -s)
```

Comment:
Try this
```yaml
    - name: Configure ssh
      run: |
        mkdir -p /root/.ssh
        chmod -R 700 /root/.ssh
        cd /root/.ssh

        printf "${{ secrets.SSH_KEY }}\n" > id_rsa
        chmod 400 id_rsa

        ssh-keyscan aur.archlinux.org > known_hosts
        chmod 644 known_hosts

        cat > config <<EOF
        Host aur aur.archlinux.org
          User aur
          HostName aur.archlinux.org
        EOF
```

Reply:
thanks! it worked!

except for 2 things which might be useful for later.
- `mkdir -p /root/.ssh` this [fails](https://github.com/The-Penguins-Club/uradhura/runs/6662879880?check_suite_focus=true#step:2:20) because the runner runs as `runner` [user](https://github.com/The-Penguins-Club/uradhura/runs/6662898005?check_suite_focus=true#step:2:24)
- `printf` fails for some reason but `echo` works :frog: